test/data/cmumps2fhir-patient7.shex

```diff
@@ -0,0 +1,587 @@
+﻿PREFIX fhir:<http://hl7.org/fhir/> 
```

Comment:
It occurs to me that it would be good to include a comment that say where this shex originally came from.